python/design_browser_history.py

```diff
@@ -0,0 +1,18 @@
+class BrowserHistory:
+
+    def __init__(self, homepage: str):
+        self.history = [homepage]
+        self.ptr = 0 
+
+    def visit(self, url: str) -> None:
+        self.ptr += 1
+        self.history = self.history[:self.ptr]
+        self.history.append(url)
+               
+    def back(self, steps: int) -> str:
+        self.ptr = max(0, self.ptr - steps)
+        return self.history[self.ptr]
+
+    def forward(self, steps: int) -> str:
+        self.ptr = min(len(self.history) - 1, self.ptr + steps)
+        return self.history[self.ptr]
```

Comment:
__[no newline at end of file]__ Can you please add a newline character at the end of the file `\n`.

Reply:
[Solved] added new line at end of file